test/unit/RouterTest.php

```diff
@@ -69,13 +69,4 @@ public function testJsonResponse()
         $this->assertIsArray(Router::getRoutes());
         $this->assertCount(5, Router::getRoutes());
     }
-
-     /** @test - Test if the route is added to the routes array */
-     public function addResource()
```

Comment:
we need this test here, the resouece test is good. Please don't remove it. put it back.

Reply:
fixed it